python/lsst/obs/base/gen2to3/repoWalker/builders.py

```diff
@@ -206,7 +210,12 @@ def build(self, parser: PathElementParser, allKeys: Dict[str, type], cumulativeK
               fileIgnoreRegEx: Optional[re.Pattern], dirIgnoreRegEx: Optional[re.Pattern]
               ) -> PathElementHandler:
         # Docstring inherited from BuilderNode.
-        return TargetFileHandler(parser=parser, translator=self._translator, datasetType=self.datasetType)
+        if self.datasetType.name == 'cpBias' or self.datasetType.name == 'cpFlat':
```

Comment:
Given the way the converter builds its datastructures, it's not trivial to do such an a configuration. This code only exists to facilitate gen2to3 conversion, so I think I'm ok with such a specialization here. @TallJimbo : any thoughts?

Reply:
I don't think this would be all that hard to do better, but given the expected shelf-life of this code and the low probability of this doing harm to other cameras, I'm not going to fight that.  We can revisit if we need more special-casing for other obs packages in the future.